sci-mathematics/nusmv/nusmv-2.6.0.ebuild

```diff
@@ -0,0 +1,83 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit cmake-utils flag-o-matic toolchain-funcs
+
+MY_P="NuSMV-${PV}"
+NUSMV_A="${MY_P}.tar.gz"
+MINISAT_A="37dc6c67e2af26379d88ce349eb9c4c6160e8543.zip"
+ZCHAFF_A="zchaff.64bit.2007.3.12.zip"
+
+DESCRIPTION="NuSMV: new symbolic model checker"
+HOMEPAGE="http://nusmv.fbk.eu/"
+SRC_URI="http://nusmv.fbk.eu/distrib/${NUSMV_A}
+		minisat? ( https://github.com/niklasso/minisat/archive/${MINISAT_A} )
+		zchaff? ( http://www.princeton.edu/~chaff/zchaff/${ZCHAFF_A} )"
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="minisat doc zchaff"
+REQUIRED_USE="|| ( minisat zchaff )"
+
+RDEPEND="dev-libs/expat"
+DEPEND="${RDEPEND}
+		doc? (
+			virtual/latex-base
+			dev-texlive/texlive-latexextra
+			app-doc/doxygen
+		)
+		>=dev-util/cmake-2.8
+		dev-lang/python:2.7
```

Comment:
depending directly on python is suspicious. Why do you need python? If you only need it at build time, use `python-any-r1`, if you need it at runtime, either `python-single-r1` or `python-r1`.